packages/terra-dev-site/src/webpack/loaderUtils/configHelpers.js

```diff
@@ -80,12 +80,20 @@ const parseExtension = filePath => {
   });
 };
 
+/**
+ * Returns a linux style file path.
+ *
+* @param {*} filePath A string representing the directory path of the file.
+*/
+const formatPath = filePath => filePath.replace(/\\/g, '/');
```

Comment:
I was thinking we should replace with [path.sep](https://nodejs.org/api/path.html#pathsep) but I think it's ok to replace with a hard-coded `/` since that is what we're using in most of the paths in generateNavigationConfig.